instructions/scivis.tex

```diff
@@ -67,30 +67,50 @@ \section{The Environment}
+  \item Using whichever techniques you want, implement a way to interactively update the transfer function to change the results of the volume rendering by manipulating the way the volume samples are mapped to color + transparency.  Implement either a widget-based or node-based transfer function editor as shown in the lecture.  See the checklist below for a more detailed list of requirements.  The transfer function is updated in the \texttt{updateTransferFunction} function.  \textbf{Important}:  Don't forget to call the \texttt{triggerTransferFunctionUpdate} function after anything has changed in your transfer function editor or otherwise the rendering will not be notified of your new transfer function.
 \end{itemize}
 
 The following is a Todo list that needs to be completed with a lab assistant to have successfully passed the lab.  \textbf{Important}:  The amount of time required for this mini-project is \emph{much} more than what is allotted in the lab sessions.  There are 8\,h of lab sessions per group, but the mini-project is scheduled for 48\,h of work in total.  If you only implement things while being in the lab itself, you will \emph{not} be able to finish the project.
 
 Please note: If some time has passed between finishing the project and showing it to the lab assistants, take some time to refamiliarize yourself with the code you have written before calling them over.  ``I don't remember, this has been a while'' is \emph{not} a good answer to any of the questions below.
 \begin{todolist}
   \item Read the \emph{entire} instructions carefully
-  \item Interact with the camera controls and explain what the step size does and how volume rendering works and how you get a color for a pixel.  For this explanation you can assume that you already have an entry and an exit point, there is no need to explain anything prior to that.  Do this on a piece of paper by drawing a sketch.
+  \item Interact with the camera controls and explain what the step size does and how volume rendering works and how you get a color for a pixel.  For this explanation you can assume that you already have an entry and an exit point, there is no need to explain anything prior to that.  Do this by drawing a sketch, preferably on a pece a paper. \textbf{Make sure you have done this sketch before presenting the lab} so that you can use it to explain how volume rendering works.
```

Comment:
```suggestion
  \item Interact with the camera controls and explain what the step size does and how volume rendering works and how you get a color for a pixel.  For this explanation you can assume that you already have an entry and an exit point, there is no need to explain anything prior to that.  Do this by drawing a sketch, preferably on a piece a paper. \textbf{Make sure you have done this sketch before presenting the lab} so that you can use it to explain how volume rendering works.
```